.github/workflows/on-pr-charts.yaml

```diff
@@ -22,8 +22,6 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-        with:
-          fetch-depth: "0"
```

Comment:
I'm not sure if this can be removed. The chart-testing-action uses the changes to find the charts which are changed. That's the default behavior unless it is specified otherwise in the ct.yaml

Reply:
good point! I removed it everywhere, but that won't work in here.